tensorflow_datasets/core/lazy_imports_lib.py

```diff
@@ -12,8 +12,8 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
 # Lint as: python3
+
```

Comment:
Why change this ? This make it inconsistent with other files.

Reply:
I am sorry for these changes , I just copy code from my local cloned repo of tfds I can fix them now
Thanks for letting me know